tests/test_playbooks/fixtures/compute_resources_role-0.yml

```diff
@@ -70,9 +70,11 @@ interactions:
     uri: https://foreman.example.org/api/compute_resources?search=name%3D%22libvirt-cr01%22&per_page=4294967296
   response:
     body:
-      string: "{\n  \"total\": 1,\n  \"subtotal\": 0,\n  \"page\": 1,\n  \"per_page\":
+      string: "{\n  \"total\": 2,\n  \"subtotal\": 1,\n  \"page\": 1,\n  \"per_page\":
```

Comment:
This recording looks like the "things" already existed...

Reply:
Yes, I did a new record against the existing Foreman after adding images. Might be that?

Anyway, I can remove the CR and do a re-record. If that's what you mean with "a clean start"?

Reply:
I removed the files and reran `make record_compute_resources_role`.

I had some issues getting the centos7-provision-nightly machine up with forklift today, so I had to change to katello 4.1. That's why some version numbers differ.